iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.h

```diff
@@ -38,9 +38,9 @@
 @property (nonatomic, readonly) TiProxy *tabGroup;
 @property (nonatomic) BOOL isMasterWindow;
 @property (nonatomic) BOOL isDetailWindow;
-@property (nonatomic) BOOL safeAreaInsetsUpdated;
+@property (nonatomic) BOOL pendingSafeAreaUpdate;
 
-- (void)processForSafeArea;
+- (bool)processForSafeArea;
```

Comment:
Use the ObjC `BOOL` type (`YES` and `NO`)

```suggestion
- (BOOL)processForSafeArea;
```